prow/cmd/deck/static/script.js

```diff
@@ -499,7 +502,7 @@ function redraw(fz) {
         builds.appendChild(r);
     }
     const jobCount = document.getElementById("job-count");
-    jobCount.textContent = "Shows " + totalJob + " job(s)";
+    jobCount.textContent = "Found " + totalJob + " jobs";
```

Comment:
LGTM, but we should consider some way to make it clear that we're not showing all of these and possibly allow the user to try to load more until their page dies (button at the bottom of the scroll list when we know we're truncating?)

Reply:
How about pagination?

Reply:
We could change the message to say something like "Showing 500/17,000 jobs."
That would make what we are showing and the total count explicit.

Reply:
that SGTM, we can revisit pagination etc. later